### PR TITLE
Executors pool size config

### PR DESCRIPTION
**Pull request by krisztiankocsis**
Rational: It would be a good idea to have an alternative configuration style for custom executors like:

executors {
  custom {
    type = fixed
    size = ${runtime.processors-x2}
  }
}

### CODE REVIEW

File: jooby-executor/src/main/java/org/jooby/exec/Exec.java [hunk 1]
```diff
@@ -360,23 +362,38 @@ private static ForkJoinWorkerThreadFactory fjwtf(final String name) {
...
+      options.put(type, config.containsKey("size") ?
+        Integer.parseInt(config.get("size").toString()) : n);
+      options.put("daemon", config.containsKey("daemon") ?
+        Boolean.parseBoolean(config.get("daemon").toString()) : daemon);
+      options.put("asyncMode", config.containsKey("asyncMode") ?
+        Boolean.parseBoolean(config.get("asyncMode").toString()) : false);
+      options.put("priority", config.containsKey("priority") ?
+        Integer.parseInt(config.get("priority").toString()) : priority);
+    } else {
+      System.out.println("Bere");
```
Comment (jknack) [inline jooby-executor/src/main/java/org/jooby/exec/Exec.java]: remove println statement

**Comment by krisztiankocsis** [comment]
Working on it.

File: jooby-executor/src/main/java/org/jooby/exec/Exec.java (hunk 1 above)
Comment (krisztiankocsis) [inline jooby-executor/src/main/java/org/jooby/exec/Exec.java]: Done

**Comment by krisztiankocsis** [comment]
Done

**Pull request merged**